Bottom: 41cce43f059f01becac52b9d0535bf54903736b4
Top:    f914c2598ee4ffa690a47afe87912cafb45b7c36
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-20 15:57:19 -0500

Refresh of update-gitlab-to-10-1

---
diff --git a/ansible-playbook/gitlab.yml b/ansible-playbook/gitlab.yml
index eb296595..4d6482cd 100644
--- a/ansible-playbook/gitlab.yml
+++ b/ansible-playbook/gitlab.yml
@@ -104,7 +104,7 @@
     # - { last: "8.15",  version: "8.16",  tag: "8.16.9-ce.0"  } # Health-check failure
     # - { last: "8.16",  version: "8.17",  tag: "8.17.8-ce.0"  } # Health-check failure
     # - { last: "8.17",  version: "9.0",   tag: "9.0.13-ce.0"  } # Health-check failure
-    # - { last: "9.0",   version: "9.1",   tag: "9.1.10-ce.0"  } 
+    # - { last: "9.0",   version: "9.1",   tag: "9.1.10-ce.0"  } # docker healthcheck introduced
     # - { last: "9.1",   version: "9.2",   tag: "9.2.10-ce.0"  }
     # - { last: "9.2",   version: "9.3",   tag: "9.3.11-ce.0"  }
     # - { last: "9.3",   version: "9.4",   tag: "9.4.7-ce.0"   }
